jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/AttributeNormalizer.java

```diff
@@ -11,7 +11,7 @@
 // ========================================================================
 //
 
-package org.eclipse.jetty.ee9.quickstart;
+package org.eclipse.jetty.util.resource;
```

Comment:
Different PR for this too

Reply:
Actually this change is for this PR.  It turns out that the `AttributeNormalizer` has for a while been assuming that a Path is available, so the main change here is to make it use the stream API for CombinedResources to generate multiple URI to resource mappings.

But as the Normalizer code was identical in ee9 and ee10, I also moved to resource to avoid duplicating the change. 

So I think the change does belong here.